tests/dbconn.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
remove dbconn tests file, as it is just testing the PHP PDO function (which will already be covered by PHP language internal tests) not any custom functionality you have created with it